### PR TITLE
Ssr

### PR DESCRIPTION
**Pull request by guilbill**
## Description

Add ssr support.

## ToDo list:

- [x] Add getServerSideProps to load the game before rendering

Peek 15-02-2021 16-29

### CODE REVIEW

**Review by djhi — COMMENTED** [review]
Not sure why you removed `_app.js` and `_document.js`. I thought they were required by NextJS

**Comment by guilbill** [comment]
It was a mistake and it is indeed required by NextJs... I've add it back in the next PR

**Review by djhi — APPROVED** [review]

File: pages/tipsy/game.jsx [hunk 1]
```diff
@@ -179,6 +189,17 @@ const Game = () => {
     );
 };
 
+Game.propTypes = {
+    currentGame: PropTypes.object,
+};
+
+export async function getServerSideProps() {
+    let game = await gameApi.newGame("Brice");
+    game = await gameApi.joinGame("Maxime", game.id);
```
Comment (zyhou) [inline pages/tipsy/game.jsx:198]: https://media1.tenor.com/images/45f94c55377f42e344019c0b13e95c3b/tenor.gif?itemid=8385316

Comment (zyhou) [inline pages/tipsy/game.jsx:198]: tenor

**Pull request merged**